frontend/tet/youth/browser-tests/pages/frontpage.ts

```diff
@@ -0,0 +1,13 @@
+import { t, Selector } from 'testcafe';
+
+class Frontpage {
+  quickSearchInput = Selector('#searchText');
```

Comment:
using class is nice and perhaps better than my function-like components-apis. However I would prefer to use [testing-library](https://testing-library.com/) -style of selectors instead of css-selectors, something like this could be done: https://dev-tester.com/fix-your-flaky-end-to-end-tests-with-testing-library/
One benefit is that it enforces better accessibiity etc, if you cannot find a good selector, then probably the element is not very accessible either :)
Anyway we can keep these selectors for now but consider to refactor later?

Reply:
Actually no need to do anything for this pr. I'm working on Helsinki lisä browser test and started to create an abstract page component class for these page model classes which sets translations and selectors.
- Here is the [root class](https://github.com/City-of-Helsinki/yjdh/blob/992e7682d0195bf931283ee63fe3c499af8f44e2/frontend/shared/browser-tests/page-models/PageComponent.ts) 
- And here is the [base class](https://github.com/City-of-Helsinki/yjdh/blob/992e7682d0195bf931283ee63fe3c499af8f44e2/frontend/benefit/applicant/browser-tests/page-model/ApplicantPageComponent.ts) per project that set translations.
- And finally a [sub class](https://github.com/City-of-Helsinki/yjdh/blob/992e7682d0195bf931283ee63fe3c499af8f44e2/frontend/benefit/applicant/browser-tests/page-model/TermsOfService.ts) for page component which has concrete selectors and actions.

Perhaps if this is a good idea, we can adapt it gradually with other projects?

Reply:
Now the [page component](https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/shared/browser-tests/page-models/PageComponent.ts) root class and others are merged to develop and can be used also in tet-browser test, if needed. But perhaps it's better to start use it later and try to get this pr in first :)